Sources/Hyperspace/Service/Network/NetworkService.swift

```diff
@@ -49,13 +49,13 @@ extension NetworkService: NetworkServiceProtocol {
         let task = session.dataTask(with: request) { [weak self] (data, response, error) in
             self?.networkActivityController?.stop()
             
-            guard let statusCode = (response as? HTTPURLResponse)?.statusCode else {
+            guard let statusCode = (response as? HTTPURLResponse)?.statusCode, let headers = (response as? HTTPURLResponse)?.allHeaderFields as? [String: String] else {
```

Comment:
Would it be cleaner to cast the response to `HTTPURLResponse` once, then access it twice?

Reply:
I can do that